pkg/adapter/adapter.go

```diff
@@ -152,20 +153,29 @@ func (a *mongoDbAdapter) makeCloudEvent(data bson.M) (*cloudevents.Event, error)
 	// Create Event.
 	event := cloudevents.NewEvent(cloudevents.VersionV1)
 
-	// Set cloud event specs and attributes. TODO: issue #43
+	// Set cloud event specs and attributes.
 	// 		ID     -> id of mongo change object
 	// 		Source -> Source environment variable.
-	// 		Type   -> type of change either insert, delete or update.
+	// 		Type   -> type of change either inserted, deleted or updated.
 	//		Data   -> data payload containing either id only for
 	//                deletion or full object for other changes.
 	change, err := utils.DecodeChangeBson(data)
 	if err != nil {
 		return nil, fmt.Errorf("error decoding bson change object: %w", err)
 	}
 	event.SetID(change.ID)
```

Comment:
let's do something like this for this one... https://github.com/google/knative-gcp/blob/11a70b6eaacd5e70b36db7da78f6a19057374ff6/pkg/schemas/v1/auditlogs.go#L46